### DISCUSSION TITLE
The migration 0_init could not be found.

### DISCUSSION THREAD

**Discussion by MarkForLoop** [prisma/prisma#23860] [Q&A] [answered]
### Question

I first pulled the database which already existed. (npx prisma db pull)
after I ran "mkdir -p prisma/migrations/0_init" and "npx prisma migrate diff \
--from-empty \
--to-schema-datamodel schema.prisma \
--script > prisma/migrations/0_init/migration.sql"

This created the migration.sql as follows
![Screenshot 2024-04-15 at 14 51 36](https://github.com/prisma/prisma/assets/91071058/66b302fa-3f44-4dba-9b90-0e60e2ebd4d1)

after I ran "npx prisma migrate resolve --applied 0_init"

which threw this error 

> Error: P3017
> 
> The migration 0_init could not be found. Please make sure that the migration exists, and that you included the whole name of the directory. (example: "20201207184859_initial_migration")

### How to reproduce (optional)

Following the [baseline](https://www.prisma.io/docs/orm/prisma-migrate/workflows/baselining#baselining-a-database) setup process

### Information about Prisma Schema, Client Queries and Environment (optional)

- IDE: VScode
- OS: macOS
- Database: PostgreSQL
- Node.js version: v20.10.0
- Prisma Version: 5.11.0

**Comment by d33con** [comment, marked as answer]
Try changing the file encoding of the migration.sql file to **UTF-8**. I followed the baseline setup process exactly and got this error, also using VSCode terminal. After some digging I found [this](https://github.com/prisma/prisma/issues/17558#issuecomment-1627716845) and the command worked for me after I changed the encoding. My migration.sql file was UTF-16 originally.

**Comment by taburetkin** [comment]
same issue. Tried to start using migrations on an exist project and followed baseline guide like in original question
but changing utf-16 to utf-8 does not help
there should be something else
VSCode
win11
PostgreSQL
Node v22.18.0
Prisma 5.19.0
